sestest.ps1

```diff
@@ -2,9 +2,20 @@
 # Convenience Launcher for sestest
 #
 
-$sestest = "$PSScriptRoot\out\sestest\netcoreapp2.0\sestest.dll"
+# Find the latest version of sestest that was compiled
+# [This will automatically find the right version, even if someone does a release 
+# build or if the output path changes e.g. becuase it's no longer netcoreapp2.0]
+$sestest = get-childitem $PSScriptRoot\out\sestest.dll -recurse | sort-object LastWriteTimeUtc | select-object -last 1
+
+if ($sesclient -eq $null) 
```

Comment:
Shouldn't this be: `if ($sestest -eq $null)` ?

Reply:
Yes. This is what happens when I rush. 😢